gatherers/sandboxed-iframes.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const Gather = require('./gather');
+
+class SandboxedIframes extends Gather {
+
+  static gather(options) {
+    const driver = options.driver;
+
+    // Be happy if there are no iframes, or every iframe in the page has a 'sandbox' attribute.
+    return driver.querySelectorAll('iframe')
```

Comment:
one issue with this is timing. We'll have to be clear that there were only sandboxed iframes at the time we tested...it's possible some were added later (or even added and removed before we got here?)